.travis.yml

```diff
@@ -1,4 +1,11 @@
 language: generic
+env:
+  - GO_VERSION=1.9
+  - GO_VERSION=rc
+matrix:
+  allow_failures:
+    - env: GO_VERSION=rc
```

Comment:
With respect to allow failures here, my thought was that we'd always leave current & next golfing enabled, and let tip be a non-blocking failure. It would give us the heads up on what's changing, and also do a bit of "good community participation" in that we're testing future releases, while also being totally automated and something that people would happen to see and fix in the natural course of a daily look at travis.